src/main/java/org/reactome/qa/doisuggester/DOISuggesterMain.java

```diff
@@ -0,0 +1,172 @@
+			// Sort them so that you'll get consistent results when testing.
+			List<GKInstance> rleList = new ArrayList<>(reactionLikeEvents);
+			InstanceUtilities.sortInstances(rleList);
+
+			// Structure for output
+			System.out.println(rleList.size() + " Reaction-like events will be checked.");
+			Map<String, Integer> rleCountsForPathways = new HashMap<>();
+			List<List<String>> records = new ArrayList<>();
+
+			for (GKInstance currentRLE : rleList)
+			{
+				// Ignore inferred RLEs
+				if (currentRLE.getReferers(ReactomeJavaConstants.inferredFrom) == null)
+				{
+					ReactionlikeEvent rle = new ReactionlikeEvent(currentRLE, dbAdaptorPrev);
+					Suggester suggester = new Suggester(rle);
+					Set<Suggestion> rleSuggestions = suggester.getSuggestion();
+					for (Suggestion s : rleSuggestions)
+					{
+						String pathway = s.getPathway();
+						rleCountsForPathways.computeIfAbsent(pathway, x -> Integer.valueOf(0));
+						for (String rleKey : s.getRlesToInstanceEdits().keySet())
+						{
+							String instanceEdits = s.getRlesToInstanceEdits().get(rleKey).stream().reduce("", (prev, instEd) -> prev + " | " + instEd );
+							List<String> record = new ArrayList<>();
+							record.add(pathway);
+							record.add(rleKey);
+							record.add(instanceEdits);
+							records.add(record);
+							rleCountsForPathways.put(pathway, rleCountsForPathways.get(pathway) + 1);
+						}
+					}
+				}
+			}
+
+			int total = 0;
+			for (Entry<String, Integer> entry : rleCountsForPathways.entrySet() )
+			{
+				System.out.println(entry.getValue() + " RLEs for " + entry.getKey());
+				total += entry.getValue();
+			}
+			System.out.println( total + " total RLEs have DOI suggestion.");
+			printOutput(records);
+		}
+		catch (PropertyHasNoValueException | PropertyNotPresentException e)
+		{
+			System.err.println("Your properties file appears to be misconfigured: " + e.getMessage());
+			e.printStackTrace();
+			System.exit(-1);
+		}
+		catch (FileNotFoundException e)
+		{
+			System.err.println("The config.properties file could not be found. Now exiting.");
+			System.exit(-1);
+		}
+		catch (SQLException e)
+		{
+			System.err.println("SQL Exception caught - most likely there was a problem connecting to the database. Maybe check your connection settings and try again?");
+			e.printStackTrace();
+			System.exit(-1);
+		}
+		catch (InvalidAttributeException e)
+		{
+			System.err.println("Error retrieving data - an attempt was made to access data via an invalid attribute.");
+			e.printStackTrace();
+			System.exit(-1);
+		}
+		catch (Exception e)
+		{
+			e.printStackTrace();
+			System.exit(-1);
+		}
+		System.out.println("All done.");
+	}
+
+	/**
+	 * Print the suggestions to a file.
+	 * @param suggestions A list of lists - each sublist is an element of a suggestion.
+	 * @throws IOException - Could be thrown if there is a problem writing the output to a file.
+	 */
+	private static void printOutput(List<List<String>> records) throws IOException
```

Comment:
The JavaDoc and method parameter look to be mismatched with one named `suggestions` and the other called `records`.